packages/react-components/src/components/MessageStatusIcon.tsx

```diff
@@ -28,7 +28,7 @@ export const MessageStatusIcon = (props: MessageStatusIconProps): JSX.Element =>
   return (
     <>
       {/* live message is used here so that aria labels are announced on mobile */}
-      {ariaLabel && <LiveMessage message={ariaLabel} ariaLive="polite" />}
+      {ariaLabel && shouldAnnounce && <LiveMessage message={ariaLabel} ariaLive="assertive" />}
```

Comment:
It looks like the change of `ariaLive` to `assertive` breaks status announcements for Safari (for me at least)

Reply:
This seems to be perculiar to safari 17.4. This works correctly on 17.4.1